tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -12,61 +13,279 @@ import (
+					Expect(err).NotTo(HaveOccurred())
+					g.Expect(res).Should(ContainSubstring("test-loadbalancer"))
+				}, "240s", "5s").Should(Succeed())
+
+				Eventually(func(g Gomega) {
+					cmd = "curl -L --insecure http://" + ip + ":" + port + "/name.html"
+					fmt.Println(cmd)
+					res, err := e2e.RunCommand(cmd)
+					Expect(err).NotTo(HaveOccurred())
+					fmt.Println(res)
+					g.Expect(res).Should(ContainSubstring("test-loadbalancer"))
+				}, "240s", "5s").Should(Succeed())
+			}
+		})
+
+		It("Verifies Ingress", func() {
+			_, err := e2e.DeployWorkload("ingress.yaml", kubeConfigFile, false)
+			if err != nil {
+				fmt.Println(err, "ingress manifest not deployed")
+				Expect(err).NotTo(HaveOccurred())
+			}
+			for _, nodeName := range serverNodenames {
+				ip, _ := e2e.FetchNodeExternalIP(nodeName)
+				cmd := "curl  --header host:foo1.bar.com" + " http://" + ip + "/name.html"
+				fmt.Println(cmd)
+
+				Eventually(func(g Gomega) {
+					res, err := e2e.RunCommand(cmd)
+					g.Expect(err).NotTo(HaveOccurred())
+					fmt.Println(res)
+					g.Expect(res).Should(ContainSubstring("test-ingress"))
+				}, "240s", "5s").Should(Succeed())
+			}
+		})
+
+		It("Verifies Daemonset", func() {
+			_, err := e2e.DeployWorkload("daemonset.yaml", kubeConfigFile, false)
+			if err != nil {
+				fmt.Println(err, "daemonset manifest not deployed")
+				Expect(err).NotTo(HaveOccurred())
+			}
+			nodes, _ := e2e.ParseNodes(kubeConfigFile, false)
+			pods, _ := e2e.ParsePods(kubeConfigFile, false)
+
+			Eventually(func(g Gomega) {
+				count := e2e.CountOfStringInSlice("test-daemonset", pods)
+				fmt.Println("POD COUNT")
+				fmt.Println(count)
+				fmt.Println("NODE COUNT")
+				fmt.Println(len(nodes))
+				g.Expect(len(nodes)).Should((Equal(count)), "Daemonset pod count does not match node count")
+			}, "420s", "10s").Should(Succeed())
+		})
+
+		It("Verifies dns access", func() {
+			_, err := e2e.DeployWorkload("dnsutils.yaml", kubeConfigFile, false)
+			if err != nil {
+				fmt.Println(err, "dnsutils manifest not deployed")
+				Expect(err).NotTo(HaveOccurred())
+			}
+			Eventually(func(g Gomega) {
+				cmd := "kubectl get pods dnsutils --kubeconfig=" + kubeConfigFile
+				res, _ := e2e.RunCommand(cmd)
+				fmt.Println(res)
+				g.Expect(res).Should(ContainSubstring("dnsutils"))
+			}, "420s", "2s").Should(Succeed())
+
+			Eventually(func(g Gomega) {
+				cmd := "kubectl --kubeconfig=" + kubeConfigFile + " exec -i -t dnsutils -- nslookup kubernetes.default"
+				fmt.Println(cmd)
+				res, _ := e2e.RunCommand(cmd)
+				fmt.Println(res)
+				g.Expect(res).Should(ContainSubstring("kubernetes.default.svc.cluster.local"))
+			}, "420s", "2s").Should(Succeed())
+		})
+
+		It("Verifies Local Path Provisioner storage ", func() {
+			_, err := e2e.DeployWorkload("local-path-provisioner.yaml", kubeConfigFile, false)
+			if err != nil {
+				fmt.Println(err, "local-path-provisioner manifest not deployed")
+				Expect(err).NotTo(HaveOccurred())
+			}
```

Comment:
This can be combined to one line
```suggestion
			Expect(err).NotTo(HaveOccurred(), "local-path-provisioner manifest not deployed")
```